tests/tests.py

```diff
@@ -1,32 +1,89 @@
-import unittest
-import sys
 import os
+import sys
+import unittest
+
+sys.path.insert(0,
+                os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
 
-sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
 
 class ImportTests(unittest.TestCase):
 
     def test_import_fft(self):
-        from dislib.fft import fft
+        pass
 
     def test_import_cascadecsvm(self):
-        from dislib.classification import CascadeSVM
+        pass
 
     def test_import_kmeans(self):
-        from dislib.cluster import KMeans
+        pass
 
     def test_import_dbscan(self):
-        from dislib.cluster import DBSCAN
-
+        pass
 
 
 class ResultsTest(unittest.TestCase):
 
     def test_cascadecsvm(self):
-        from dislib.classification import CascadeSVM
+        pass
+
+
+class DBSCANTest(unittest.TestCase):
+
+    def test_basic(self):
+        from sklearn.datasets import make_blobs, make_circles, make_moons
+        from dislib.cluster import DBSCAN
+        from sklearn.preprocessing import StandardScaler
+        from dislib.data import load_data
+        import numpy as np
+
+        n_samples = 1500
+
+        # Test blobs
+        x, y = make_blobs(n_samples=n_samples, n_features=2, random_state=8)
+        dbscan = DBSCAN(grid_dim=1, eps=.3)
+        x = StandardScaler().fit_transform(x)
+        dataset = load_data(x=x, y=y, subset_size=300)
+        dbscan.fit(dataset)
+        n_clusters = np.unique(dbscan.labels_)
+        n_clusters = n_clusters[n_clusters >= 0].size
+        self.assertEqual(n_clusters, 3)
```

Comment:
each self.assertSomething, should be a separate test with a name explaining what it does. E.g: def test_number_of_clusters